### PR TITLE
wrapper script for copy-from' command

### PR DESCRIPTION
**Pull request by aviadl**
Will also set file metadata into context, if key is provided
Need to validate that the depends on parameters are set correctly
also need to validate that the fromversion is set correctly

### CODE REVIEW

**Comment by aviadl** [comment]
Need to make script accept a list of files (csv or array)
Need to allow limiting to a specific instance
Need to allow fetching from multiple instances

File: Scripts/script-RACopyFrom.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  Copy a file using remote access integration.
+  Will place file attributes on contextKey param:
+  Assuming contextKey parameter is foo, the following keys will be set:
+  foo/MD5
+  foo/SHA1
+  foo/SHA256
+  foo/sha512
+  foo/SSDeep
+  foo/Type
+enabled: true
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: Not needed. Add "system: true" instead

File: Scripts/script-RACopyFrom.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      setContext(args.contextKey+"/SHA1", res.FileMetadata.SHA1);
+      setContext(args.contextKey+"/SHA256", res.FileMetadata.SHA256);
+      setContext(args.contextKey+"/SHA512", res.FileMetadata.SHA512);
+      setContext(args.contextKey+"/SSDeep", res.FileMetadata.SSDeep);
+      setContext(args.contextKey+"/Type", res.FileMetadata.Type);
+  }
+
+  return res;
+type: javascript
+tags: []
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: Add tags

File: Scripts/script-RACopyFrom.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+commonfields:
+  id: RACopyFrom
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: I suggest renaming to RemoteaccessCopyFrom. Much more descriptive and not that long

File: Scripts/script-RACopyFrom.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+name: RACopyFrom
+fromversion: 2.0.1
+script: |-
+  var files = executeCommand("copy-from", {"file":args.filePath})
+  var err = null;
+  var res = null
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: Missing some ; (in a few other lines too)

File: Scripts/script-RACopyFrom.yml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- name: filePath
+  required: true
+  default: true
+  description: File to copy to war room
+- name: contextKey
+  description: Context key to store file details on
+scripttarget: 0
+dependson: 
+  must:
+  - remoteaccess
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: These are commands, so it needs to be "copy-from"

File: Scripts/script-RACopyFrom.yml [hunk 6]
```diff
@@ -0,0 +1,54 @@
+commonfields:
+  id: RACopyFrom
+name: RACopyFrom
+fromversion: 2.0.1
+script: |-
+  var files = executeCommand("copy-from", {"file":args.filePath})
```
Comment (liorkol) [inline Scripts/script-RACopyFrom.yml]: Script should take and forward "using" arg. Pretty quick and worth it for the added power

**Comment by aviadl** [comment]
@gshriki you have modified this script on you end.
Please validate that it fits the comments @liorkol has wrote and place it in this PR

**Comment by meirwah** [comment]
closing this...

**Pull request closed without merging**